README.md

```diff
@@ -34,71 +34,85 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+- Semantic HTML is the concept of using more descriptive tags while marking up your html document. There are tags that do the same things but certain tags have a more descriptive name. This is useful for when you are working with other teammates who also have to read what you have wrote and may even be useful to yourself. The difference between a semantic tag and a non-semantic tag is the difference between a cardboard box and a box with a label. Both function the same but you will have an idea of what is in the labeled box before you even look in it because of the label.
```

Comment:
Good definition! Semantic HTML give meaning to your code to Search Engines/Browsers. As you progress through lambda, these things will become second nature